Added a paragraph about using only Annotations as mapping type (#572)
overblog/GraphQLBundle: docs/annotations/index.md

This will load all annotated classes in `%kernel.project_dir%/src/GraphQL` into the schema.

> ## Use Annotations as your only Mapping

Reviewer: ```suggestion
## Using Annotations as your only Mapping
```

> If you only use annotations as mappings you need to add an empty RootQuery type.

Reviewer 2: ```suggestion
If you only use annotations as mappings you need to add an empty `RootQuery` type.
```

Your config should look like this:
```yaml
# config/packages/graphql.yaml
overblog_graphql:
  definitions:
    schema:
      query: RootQuery
    mappings:
      types:
        - type: annotation
          dir: "%kernel.project_dir%/src/GraphQL"
          suffix: ~
```

> Your RootQuery class should look like this:

Reviewer 2: ```suggestion
Your `RootQuery` class should look like this:
```

```php
namespace App\GraphQL\Query;

/**
 * @GQL\Type
 */
class RootQuery
{
}
```

> If you use mutations, you need a RootMutation aswell.

Reviewer 2: ```suggestion
If you use mutations, you need a `RootMutation` type as well.
```
